cmd/utils/interfaces.go

```diff
@@ -0,0 +1,69 @@
+// Copyright © 2022 SECO Mind Srl
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/astarte-platform/astarte-go/interfaces"
+
+	"github.com/spf13/cobra"
+)
+
+var interfacesCmd = &cobra.Command{
+	Use:   "interfaces",
+	Short: "Utility operations on Astarte Interfaces",
+}
+
+var validateInterfaceCmd = &cobra.Command{
+	Use:   "validate <interface_file>",
+	Short: "Validates an interface",
+	Long: `Checks whether the provided JSON file is a valid Astarte Interface.
+
+Returns 0 and does not print anything if the interface is valid, returns 1 and prints an error message if it isn't.`,
+	Example: `  astartectl utils interfaces validate com.my.Interface.json`,
+	Args:    cobra.ExactArgs(1),
+	RunE:    validateInterfaceF,
+}
+
+func init() {
+	UtilsCmd.AddCommand(interfacesCmd)
+
+	interfacesCmd.AddCommand(
+		validateInterfaceCmd,
+	)
+}
+
+func validateInterfaceF(command *cobra.Command, args []string) error {
+	interfacePath := args[0]
+	interfaceFile, err := ioutil.ReadFile(interfacePath)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Cannot read %s\n", interfacePath)
+		os.Exit(1)
+		return nil
+	}
+
+	var interfaceBody interfaces.AstarteInterface
+	if err = json.Unmarshal(interfaceFile, &interfaceBody); err != nil {
```

Comment:
Does astarte-go provide any kind of meaningful error? If so, it would be nice to print it